qs.ts

```diff
@@ -1,11 +1,19 @@
+interface ParseOptions {
+  sep?: string;
+  eq?: string;
+}
+
 export class qs {
-  static parse(query: string, sep: string = "&"): Object {
+  static parse(
+    query: string,
+    { sep = "&", eq = "=" }: ParseOptions = {}
+  ): Record<string, string> {
     let segmentation = query.split(sep);
 
     return segmentation.reduce(
       (result: Record<string, string>, str: string) => {
-        let key: string = str.split("=")[0];
-        let value: string = str.split("=")[1];
+        let key: string = str.split(eq)[0];
+        let value: string = str.split(eq)[1];
```

Comment:
What if there's multiple times `eq`? `a=b=c`

Reply:
```node
➜  deno-qs git:(master) node                                                                                                                           │·······································
> querystring.parse('a=c')                                                                                                                             │·······································
[Object: null prototype] { a: 'c' }                                                                                                                    │·······································
> querystring.parse('a=c=b')                                                                                                                           │·······································
[Object: null prototype] { a: 'c=b' }                                                                                                                  │·······································
```

I'll add a test on it to be sure we return the same thing.